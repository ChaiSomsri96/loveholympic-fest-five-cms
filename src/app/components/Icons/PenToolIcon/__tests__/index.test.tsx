import * as React from 'react';
import { render } from '@testing-library/react';

import { PenToolIcon } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<PenToolIcon  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<PenToolIcon />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
