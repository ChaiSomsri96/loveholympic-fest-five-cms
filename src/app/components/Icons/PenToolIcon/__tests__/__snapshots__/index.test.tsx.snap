// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PenToolIcon  /> should match snapshot 1`] = `
<svg
  fill="none"
  height="20"
  viewBox="0 0 20 20"
  width="20"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M10 15.8333L15.8333 10L18.3333 12.5L12.5 18.3333L10 15.8333Z"
    stroke="#FFFCFE"
    stroke-linecap="round"
    stroke-linejoin="round"
  />
  <path
    d="M15 10.8333L13.75 4.58332L1.66663 1.66666L4.58329 13.75L10.8333 15L15 10.8333Z"
    stroke="#FFFCFE"
    stroke-linecap="round"
    stroke-linejoin="round"
  />
  <path
    d="M1.66663 1.66666L7.98829 7.98832"
    stroke="#FFFCFE"
    stroke-linecap="round"
    stroke-linejoin="round"
  />
  <path
    d="M9.16667 10.8333C10.0871 10.8333 10.8333 10.0871 10.8333 9.16667C10.8333 8.24619 10.0871 7.5 9.16667 7.5C8.24619 7.5 7.5 8.24619 7.5 9.16667C7.5 10.0871 8.24619 10.8333 9.16667 10.8333Z"
    stroke="#FFFCFE"
    stroke-linecap="round"
    stroke-linejoin="round"
  />
</svg>
`;
